scripts/criu-ns

```diff
@@ -80,8 +80,20 @@ def run_criu(args):
     Spawn CRIU binary
     """
     print(sys.argv)
-    os.execlp('criu', *['criu'] + args)
-    raise OSError(errno.ENOENT, "No such command")
+
+    if "--criu-binary" in args:
+        try:
+            path = args[args.index("--criu-binary") + 1]
+            args.remove("--criu-binary")
+            args.remove(path)
```

Comment:
This is kinda risky e.g. if path string occurs twice in args, you can remove wrong path. I would prefer removing by index (list.pop(index)) as more robust solution.

Reply:
Done, thanks!